Pomegranate.prefab

```diff
@@ -0,0 +1,26 @@
+{
+	    "parent": "engine:iconItem",
```

Comment:
Remove tab spacing in all lines of all files. We want only 4 spaces for indentation.
You might want to configure your intellij settings for this.